api/v1/README.md

```diff
@@ -8,6 +8,7 @@
 - [Namespaces](#namespaces)
   - [GET](#get-namespaces)
 - [Vulnerabilities](#vulnerabilities)
+  - [List](#get-namespacesnsnamevulnerabilities)
```

Comment:
I wish there was a better way to put this, because I was using HTTP verbs for all of these and technically this is a `GET`